lib/features/interaction-events/BpmnInteractionEvents.js

```diff
@@ -68,6 +68,12 @@ BpmnInteractionEvents.prototype.createParticipantHit = function(element, gfx) {
   // remove existing hits
   this._interactionEvents.removeHits(gfx);
 
+  // add body hit
+  this._interactionEvents.createBoxHit(gfx, 'no-move', {
```

Comment:
Incredible, actually we had all infrastructure in place already. Know your tooling I guess (which I forgot about) :smile_cat:.